cmd/root.go

```diff
@@ -131,6 +131,16 @@ func initPopeyeFlags() {
 		"Specify to which S3 bucket you want to save the output file",
 	)
 
+	rootCmd.Flags().StringVarP(flags.S3Region, "s3-region", "",
+		"",
+		"Specify to which S3 region you want to save the output file",
```

Comment:
We should change this description to something like "Specify the storage region if needed when the s3-bucket option is enabled" and likewise for s3-endpoint.

Reply:
fixed :)